### ISSUE TITLE
Deno lint rule no-undef and Temporal: Temporal is not defined

### ISSUE THREAD

**Issue by maximilian-hammerl** [denoland/deno_lint#1336]
There seems to be no way to forward unstable flags to deno lint.
`deno lint --unstable-temporal` and `deno lint --config deno.json` with `"unstable": ["temporal"],` do not work.

### Lint Name

no-undef

### Code Snippet

```ts
Temporal.Now.instant()
```

### Expected Result

deno lint understand, that Temporal is defined

### Actual Result

error[no-undef]: Temporal is not defined

### Version

deno 1.46.3 (stable, release, aarch64-apple-darwin)
v8 12.9.202.5-rusty
typescript 5.5.2
